module/src/components/modal/modal.hooks.tsx

```diff
@@ -63,7 +63,9 @@ export const useModalLayerPromise = <T, TArg = unknown>(
 
         const rejectPromise = () => {
           close();
-          reject();
+          if (reject) {
+            reject();
+          }
```

Comment:
I'm a `reject?.()` guy these days, but each to their own.

Reply:
I think the `if` is way more readable but it does look like the rest of the code uses your way so I'll change it to bring it inline